src/styles/App/_events-by-category-page.scss

```diff
@@ -0,0 +1,87 @@
+.events-by-category-page {
+  > h2 {
+    font-size: 2.3rem;
```

Comment:
you only have one h2, you don't need to select the direct h2 from the parent, h2 should be enough.